config.go

```diff
@@ -740,6 +803,11 @@ func getConfig() *types.Configuration {
 	c.Mattermost.MessageFormatTemplate = getMessageFormatTemplate("Mattermost", c.Mattermost.MessageFormat)
 	c.Googlechat.MessageFormatTemplate = getMessageFormatTemplate("Googlechat", c.Googlechat.MessageFormat)
 	c.Cliq.MessageFormatTemplate = getMessageFormatTemplate("Cliq", c.Cliq.MessageFormat)
+	c.OTLP.Traces.Endpoint = strings.TrimSpace(c.OTLP.Traces.Endpoint)
+	if c.OTLP.Traces.TraceIDHash != "" {
+		c.OTLP.Traces.TraceIDHashTemplate = getMessageFormatTemplate("OTLP", c.OTLP.Traces.TraceIDHash).Option("missingkey=zero")
+	}
```

Comment:
Is it really a requested feature? Can't we just enforce it by our own? It's not clear to me which is the benefit

Reply:
I found it to be safer to let the gotemplate engine fallback to `""` instead of the verbatim `"<no value>"` if that option is not set, see https://pkg.go.dev/text/template#Template.Option.